src/ui/Navbar.fs

```diff
@@ -4,67 +4,125 @@ open Feliz
+                            prop.text "About Us"
+                            prop.onClick (fun _ -> Router.navigatePath "/about")
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
 
                     NavbarItem [
                         Link [
                             prop.text "Contact"
-                            color "foreground"
                             prop.onClick (fun _ -> Router.navigatePath "/contact")
-                            prop.style [ style.cursor "pointer" ]
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
                 ]
             ]
 
             NavbarContent [
                 justify "end"
+                prop.style [ length.px 40 |> style.gap ]
                 prop.children [
                     NavbarItem [
-                        Button [ color "primary"; prop.custom ("variant", "flat"); prop.text "Cart" ]
+                        img [
+                            prop.src <| import "default" "./assets/Cart.svg"
+                            prop.onClick (fun _ -> Router.navigatePath "/")
+                            prop.style [
+                                length.px 21 |> style.height
+                                style.cursor "pointer"
+                                (if isMenuOpen then 0 else 100) |> style.filter.invert
+                            ]
+                        ]
+                    ]
+
+                    NavbarMenuToggle [
+                        prop.ariaLabel <| if isMenuOpen then "Close menu" else "Open menu"
+                        prop.className "sm:hidden"
+                        prop.style [ (if isMenuOpen then "black" else "white") |> style.color ]
                     ]
                 ]
             ]
+
+            NavbarMenu [
+                prop.style [ style.paddingTop 0; style.backgroundColor "#F7F4F0" ]
+                menuItems
+                |> List.indexed
+                |> List.map (fun (index, item) ->
+                    NavbarMenuItem [
+                        prop.key $"{item}-{index}"
+                        prop.children [
+                            Link [
+                                color
+                                <| if index = 2 then "foreground"
+                                   else if index = menuItems.Length - 1 then "danger"
+                                   else "foreground"
+                                prop.className "w-full"
+                                prop.style [
+                                    style.fontFamily "the-seasons"
+                                    length.perc 320 |> style.fontSize
+                                    length.fitContent |> style.height
+                                    (if item = "Shop" then 120 else 30) |> style.paddingTop
```

Comment:
This is hardcoded behaviour! BAD! If what you _ACTUALLY_ mean here is to put extra padding on the first item, then simply put that as top padding on the parent. If you really want it to be part of the first item's hitbox, then just check the index instead of assuming that the shop will always be the first item and have that exact spelling and capitalisation.

Reply:
When I put padding on the parent it ruined the animation transition, that's why I changed it. But yeah using the index here would be a good idea.